index.html

```diff
@@ -25,15 +25,14 @@ <h2 class="project-tagline">ETL programming in Python</h2>
         <section class="main-content">
             <!-- Introduction Section -->
             <h1><a id="pygrametl---etl-programming-in-python" class="anchor" href="#pygrametl---etl-programming-in-python" aria-hidden="true"><span class="octicon octicon-link"></span></a>pygrametl - ETL programming in Python</h1>
-            <p>pygrametl (pronounced py-gram-e-t-l) is a Python framework that provides commonly used functionality for the development of Extract-Transform-Load (ETL) processes. It is open-source and released under a 2-clause BSD license.</p>
+            <p>pygrametl (pronounced py-gram-e-t-l) is a Python framework that provides functionality commonly used when developing Extract-Transform-Load (ETL) programs. It is fully open-source and released under a 2-clause BSD license. As shown in the figure below, an ETL program developed using pygrametl consists of SQL statements that create the data warehouse's schema (i.e., dimension tables, fact tables, and indices) and standard Python code that implements the ETL program itself.</p>
+            <p><center><img src="assets/etl-with-pygrametl.svg" alt="ETL process using pygrametl"></center></p>
 
-            <p>When using pygrametl, the developer codes the ETL process in Python code. This turns out to be very efficient, also when compared to drawing the process in a graphical user interface (GUI) like Pentaho Data Integration. It supports both CPython and Jython so existing Java code and JDBC drivers can be used in the ETL program.</p>
+            <p>Defining the data warehouse's schema using SQL and implementing the ETL program itself using standard Python turns out to be very efficient and effective, even when compared to drawing the program in a graphical user interface like Apache Hop or Pentaho Data Integration. pygrametl supports CPython and Jython so both existing Python code that use native extensions models and PEP 249 connectors and JVM-based code that use JDBC drivers can be used in the ETL program.</p>
```

Comment:
Similar comment about defining the DW schema...

2 x "code that use" -> "code that uses"

Reply:
Unclear what you mean by _"Similar comment about defining the DW schema"_, here it is directly stated that the schema is defined using SQL and the ETL program using Python.

Fixed 2 x "code that use" -> "code that uses".

Reply:
yes, no confusion here :-)